server/middleware/setUpMultipartFormDataParsing.ts

```diff
@@ -30,7 +32,17 @@ export default function setUpMultipartFormDataParsing(allowMultiple = false): Ro
   })
 
   const fileFilter = (req: FileUploadRequest, file: Express.Multer.File, cb: multer.FileFilterCallback) => {
-    cb(null, true)
+    switch (fileUploadMode) {
+      case 'videoUpload':
+        if (file.mimetype.startsWith('video/')) {
+          cb(null, true)
```

Comment:
What does null here refer to, what does true refer to here?

Reply:
So I'm assuming these are documented parts of multer. FileFilterCallback rather than our code?